gnosis/gp-ui
# 917 rename xdai to gc

# Summary

Closes #917 

- [x] Replace **XDAI** to **GNOSIS CHAIN** on Header Logo
- [x] Substitute `/xdai/...` to `/gc/...` on URL
- [x] Replace **xDai** name from `UserDetailsTable` when no orders and found on other networks

## To Test

1. Header Logo
  Go to https://pr918--gpui.review.gnosisdev.com/gc and verify Header
2. Substitute `xdai` for `gc`   
  Try to use `xdai` URL with https://pr918--gpui.review.gnosisdev.com/xdai
3. Replace **xDai** from `UserDetailsTable`
  Go to UserDetails with address on `mainnet` [0xFF714b8b0e2700303eC912BD40496C3997ceEa2b](https://pr918--gpui.review.gnosisdev.com/address/0xFF714b8b0e2700303eC912BD40496C3997ceEa2b)
